src/containers/Header/Help/HelpButton.js

```diff
@@ -7,18 +7,30 @@ import {
 	NavLink,
 	Button, Card, CardHeader, CardBody
 } from 'reactstrap';
+import { Spinner } from 'asab_webui_components';
+
 
 import './HelpButton.scss';
 
 export default function HelpButton() {
 	const {t} = useTranslation();
 
 	const [modal, setModal] = useState(false);
+	const [isIframeLoading, setIsIframeLoading] = useState(true);
 
 	const path = useSelector(state => state?.header.helpPath);
 	if (path == undefined) return null;
 
-	const toggle = () => setModal(!modal);
+	const toggle = () => {
```

Comment:
@JanMajer86 

```
const toggle = () => {
  setModal(prev => {
    const newModal = !prev;
    if (newModal) {
      setIsIframeLoading(true);
    }
    return newModal;
  });
}
```

I think this is more effective when setting the loading state, since it is triggered only when the modal is opened. Please test this approach (if it will be suitable)